quine/src/main/scala/com/thatdot/quine/app/ingest/KafkaSrcDef.scala

```diff
@@ -68,6 +75,18 @@ object KafkaSrcDef {
         AUTO_OFFSET_RESET_CONFIG -> autoOffsetReset.name,
         SECURITY_PROTOCOL_CONFIG -> securityProtocol.name
       )
+
+    // Configure kafka consumer with SASL auth if exists
+    saslAuthentication match {
+      case Some(KafkaSaslAuthentication.Plain(jaasConfig, saslMechanism)) =>
+        consumer
+          .withProperties(
+            "sasl.jaas.config" -> jaasConfig,
+            "sasl.mechanism" -> saslMechanism
```

Comment:
We've been using constants provided by the Kafka library for these strings, e.g. the `AUTO_OFFSET_RESET_CONFIG` and `SECURITY_PROTOCOL_CONFIG`
These two strings are available as the constants `SASL_JAAS_CONFIG` and `SASL_MECHANISM`, which can be imported from `org.apache.kafka.common.config.SaslConfigs`. Or maybe we should just use strings throughout for these, instead of using the constants?

Reply:
For the life of me I couldn't seem to find those constants in kafka, thanks for the import path. I think keeping to the usage of constants makes sense (unless we want to go the route of just allowing users to pass their own kafka properties as Map[String, String], then maybe just stick to strings)